src/tutorial-panel/components/tutorial-list/index.tsx

```diff
@@ -101,7 +83,6 @@ export default function TutorialList({
                   />
                 ))}
               </ul>
-              {/* {filteredTutorials.length === 0 && searchTerm && <Box>No tutorials match this search filter.</Box>} */}
             </>
```

Comment:
After removing stuff, this fragment is unneeded (let's slim down this as much as we can)